zuul.d/ansible-cloud-jobs.yaml

```diff
@@ -473,6 +473,67 @@
     run: playbooks/ansible-cloud/k8s/molecule.yaml
     required-projects:
       - name: github.com/ansible/ansible
+        override-checkout: devel
+      - name: github.com/ansible-collections/kubernetes.core
+    timeout: 5400
+    nodeset: centos-7-4vcpu
+    vars:
+      ansible_collections_repo: github.com/ansible-collections/kubernetes.core
+      ansible_test_python: 3.6
+      ansible_test_command: integration
+      ansible_test_collections: true
+      ansible_test_venv_path: "~/venv"
+
+- job:
+    name: ansible-test-molecule-kubernetes-core-stable-2.9
+    dependencies:
+      - name: build-ansible-collection
+    pre-run:
+      - playbooks/ansible-cloud/k8s/pre.yaml
+    run: playbooks/ansible-cloud/k8s/molecule.yaml
+    required-projects:
+      - name: github.com/ansible/ansible
+        override-checkout: stable-2.9
+      - name: github.com/ansible-collections/kubernetes.core
+    timeout: 5400
+    nodeset: centos-7-4vcpu
+    vars:
+      ansible_collections_repo: github.com/ansible-collections/kubernetes.core
+      ansible_test_python: 3.6
+      ansible_test_command: integration
+      ansible_test_collections: true
+      ansible_test_venv_path: "~/venv"
+
+- job:
+    name: ansible-test-molecule-kubernetes-core-stable-2.10
```

Comment:
I think we can just skip 2.10. It's not an LTS and it's superseded by 2.11 now.

Reply:
ok will do it.